src/public/map.js

```diff
@@ -380,17 +410,29 @@ $(function filterDate() {
     }
   });
 
-  $('input[name="datefilter"]').on('apply.daterangepicker', function apply(ev, picker) {
-    $(this).val(picker.startDate.format('MM/DD/YYYY') + ' - ' + picker.endDate.format('MM/DD/YYYY'));
+  $('span[name="calendar"]').daterangepicker({
+      autoUpdateInput: false,
+      locale: {
+          cancelLabel: 'Clear'
+      }
+  });
+
+  $('input[name="datefilter"]').on('apply.daterangepicker', function (ev, picker) {
+
+      $(this).val(picker.startDate.format('MM/DD/YYYY') + ' - ' + picker.endDate.format('MM/DD/YYYY'));
```

Comment:
Feel free to ignore me again 😅 

Line 422 is over-indented, as are certain lines in the next few blocks of code..